convert.py

```diff
@@ -1,4 +1,22 @@
-def utf_len(s: bytes) -> int:
+"""
+docstring
+"""
 
 
-    return 0
\ No newline at end of file
+def utf_len(string: bytes) -> int:
+    """
+
+    :param s:
+    :return:
+    """
+    new_st = str(string)[2:-1].split()
+    count = 0
+    for i in range(len(new_st)):
```

Comment:
! Checking of string representation of byte is not enough for correct solution. Try to use bit operators